api/manifest/api.go

```diff
@@ -57,40 +61,34 @@ func getGitHubReleaseAssetForSHA(aeAPI shared.AppEngineAPI, sha string) (fetched
 	}
 	var release *github.RepositoryRelease
 	releaseTag := "latest"
-	fetchedSHA = sha
 	if shared.IsLatest(sha) {
 		// Use GitHub's API for latest release.
 		release, _, err = client.Repositories.GetLatestRelease(aeAPI.Context(), shared.WPTRepoOwner, shared.WPTRepoName)
 	} else {
-		q := fmt.Sprintf("SHA:%s user:web-platform-tests repo:wpt", sha)
+		q := fmt.Sprintf("SHA:%s repo:web-platform-tests/wpt", sha)
```

Comment:
This is a drive-by change. The `repo:<user>/<repo>` syntax is documented whereas the combination of `user:` and `repo:` is not (but also works). I prefer to use the documented syntax.

https://help.github.com/en/github/searching-for-information-on-github/searching-issues-and-pull-requests#search-within-a-users-or-organizations-repositories